cmd/skopeo/main.go

```diff
@@ -85,30 +75,46 @@ func createApp() (*cli.App, *globalOptions) {
 			Usage:       "use `VARIANT` instead of the running architecture variant for choosing images",
 			Destination: &opts.overrideVariant,
 		},
-		cli.DurationFlag{
-			Name:        "command-timeout",
-			Usage:       "timeout for the command execution",
-			Destination: &opts.commandTimeout,
+		cli.StringFlag{
+			Name:        "policy",
+			Usage:       "Path to a trust policy file",
+			Destination: &opts.policyPath,
 		},
 		cli.StringFlag{
 			Name:        "registries-conf",
 			Usage:       "path to the registries.conf file",
 			Destination: &opts.registriesConfPath,
 			Hidden:      true,
 		},
+		cli.StringFlag{
+			Name:        "registries.d",
+			Usage:       "use registry configuration files in `DIR` (e.g. for container signature storage)",
+			Destination: &opts.registriesDirPath,
+		},
+		cli.GenericFlag{
+			Name:   "tls-verify",
+			Usage:  "require HTTPS and verify certificates when talking to container registries (defaults to true)",
+			Hidden: true,
+			Value:  newOptionalBoolValue(&opts.tlsVerify),
+		},
+		cli.StringFlag{
+			Name:        "tmpdir",
+			Usage:       "directory used to store temporary files",
```

Comment:
`s/temporary/big temporary/`? (Both here and in the man page.)

Reply:
That means nothing to the user.

Reply:
OTOH the current version is just incorrect. The difference can be explained in the man page for those who care.

\*shrug\* if someone else wants to merge this as is, feel free.